src/track/track.h

```diff
@@ -5,22 +5,21 @@
 #include <QObject>
 #include <QUrl>
 
+#include "util/memory.h"
+#include "util/sandbox.h"
+
+class Track;
```

Comment:
Please move these declarations after the #include directives. Otherwise you are relying on some some order which you shouldn't. Alternatively move these declarations into a separate header file that we can include explicitly.